pkg/collector/corechecks/oracle-dbm/config/config.go

```diff
@@ -234,6 +235,15 @@ func NewCheckConfig(rawInstance integration.Data, rawInitConfig integration.Data
 		}
 	}
 
+	/*
+	 * `instant_client` is deprecated but still supported to avoid a breaking change
+	 * `oracle_client` is a more appropriate naming because besides Instant Client
+	 * the Agent can be used with an Oracle software home.
+	 */
+	if instance.InstantClient {
```

Comment:
can we emit a warning log message here? something like "`instance_client` is deprecated and will be removed in future versions. please use `oracle_client`".

Reply:
done